client/src/components/Reservations.jsx

```diff
@@ -3,27 +3,39 @@ import ReactDOM from 'react-dom';
 class Reservations extends React.Component {
     constructor(props) {
         super(props);
-
+        console.log(props.state)
     }
     render() {
         return (
             <div id="reservations">
-                {/* <div id="dorms">
-                    <div className="header">Dorm Beds<br/><p>Prices are per room</p></div>
-                    <div className="header"><p>Average price per night</p></div>
-                    <div className="header"><p>Rooms</p></div>                    
-                </div> */}
                 <table id="dorms2">
                     <thead>
-                        <th id="header">
-                            <td className="dormName"><h4>Dorm Beds</h4>
+                        <tr id="header">
+                            <th className="dormName"><h4>Dorm Beds</h4>
                                 <span>Prices are per room</span>
-                            </td>
-                            <td className="averagePrice">Average price per night</td>
-                            <td className="countRooms">Rooms</td>
-                        </th>
+                            </th>
+                            <th className="averagePrice">Average price per night</th>
+                            <th className="countRooms">Rooms</th>
+                        </tr>
                     </thead>
                     <tbody>
+                        {console.log('yeet boi', this.props.state.hotelRooms.rooms)}

```

Comment:
yeeetttt

Reply:
Try not to have `console.log`'s in your PR's. They're great for dev and debugging, but shouldn't end up on your github if it's avoidable.